code/_core/obj/item/fishing/line.dm

```diff
@@ -0,0 +1,54 @@
+/obj/item/fishing/line
+	name = "fishing line"
+	desc = "damn bro go for the lottery today"
+	icon = 'icons/obj/item/fishing/line.dmi'
+	drop_sound = 'sound/items/drop/scrap.ogg'
+	size = SIZE_0
+	weight = 0.5
+	value = 420
+	var/lavaproof = 0
+	var/break_on_failure = 0
+	var/break_on_success = 0
+	color = "#FFFFFF"
+
+
+
+/obj/item/fishing/line/proc/get_line()

```

Comment:
Proc like this that just returns src and nothing else is not needed.

Reply:
fixed for next commit